packages/dx-react-scheduler-material-ui/src/templates/appointment/vertical-appointment.jsx

```diff
@@ -36,29 +51,41 @@ const VerticalAppointmentBase = ({
   data,
   children,
   className,
+  recurringIconComponent: RecurringIcon,
   ...restProps
-}) => (
-  children || (
+}) => {
+  const repeat = !!data.rRule;
+  return (
+    children || (
     <div className={classNames(classes.content, className)} {...restProps}>
-      <div className={classes.title}>
-        {data.title}
-      </div>
-      <div className={classes.textContainer}>
-        <div className={classes.time}>
-          {moment(data.startDate).format('h:mm A')}
-        </div>
-        <div className={classes.time}>
-          {' - '}
+      <div className={repeat ? classes.recurContainer : classes.container}>
```

Comment:
Can we keep names consistent? I mean `recurringIconComponent` but `recurContainer`.

Reply:
recurContainer -> recurringContainer?